src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -155,6 +155,13 @@ const setFirstMultipleAnswerItemRef = (
   }
 };
 
+watch(() => props.touched, (isTouched) => {
+  if (isTouched) {
+    return;
+  }
+  invalidMultipleChoicesItemRefs.clear();
```

Comment:
Is there any rule in the CL's linter or why do you wrap the `return` in parenthesis? Couldn't it be like these two lines
```
if (isTouched) return;
invalidMultipleChoicesItemRefs.clear();
```
instead of 4? 😄

Reply:
or even make it a one-liner and write:
```
 if (!isTouched) invalidMultipleChoicesItemRefs.clear();
```

Reply:
done